src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/MagicSugar.java

```diff
@@ -23,6 +26,11 @@ public MagicSugar(Category category, SlimefunItemStack item, RecipeType recipeTy
 	public ItemInteractionHandler getItemHandler() {
 		return (e, p, item) -> {
 			if (isItem(item)) {
+				if(e.getClickedBlock() != null) {
+					String target = BlockStorage.checkID(e.getClickedBlock());
+					if ("ANCIENT_PEDESTAL".equals(target) || "ANCIENT_ALTAR".equals(target))
+						return false;
```

Comment:
This should return true.

True means "Yes this Item was me".
With false you are telling Slimefun to keep looking for items that match...